content/en/blog/2024/prom-and-otel/index.md

```diff
@@ -0,0 +1,564 @@
+---
+title: Prometheus and OpenTelemetry: Better Together
+linkTitle: Prometheus and OpenTelemetry
+date: 2024-03-10
+author: 
+  [Reese Lee](https://github.com/reese-lee) (New Relic),
+  [Adriana Villela](https://github.com/avillela) (ServiceNow)
+draft: true # TODO: remove this line once your post is ready to be published
+# canonical_url: http://somewhere.else/ # This will be added in future
+---
+
+![Image of a Greek god holding a torch with the Prometheus logo, and OTel logo](Prom-and-otel-logos.png)
+
+Tools such as [Prometheus](https://prometheus.io/) and [OpenTelemetry](https://opentelemetry.io/docs/what-is-opentelemetry/) help us maintain the health, performance, and 
+availability of our complex distributed systems. Both are open source projects 
+under the [Cloud Native Computing Foundation (CNCF)](https://www.cncf.io/) 
+umbrella – but what role does each play in observability? 
+
+OpenTelemetry (OTel for short), is a vendor-neutral open standard for 
+instrumenting, generating, collecting, and exporting telemetry data. Prometheus 
+is a fixture of the observability landscape, widely relied upon for monitoring 
+and alerting within organizations. 
+
+While both Prometheus and OTel emit metrics, there is a lot to cover on the 
```

Comment:
i'd suggest to start with the most important distinction between otel and prom - otel is about instrumenting/exporting, and is *NOT* a backend to store metrics (or any telemetry). Where prometheus is all about *THE* place to store metrics, but it also include clients as well.

Reply:
Ah yes, of course. I modified the "Prometheus" section:
* Renamed it to "OTel vs Prometheus"
* Added this:

>Since OTel is primarily focused on the instrumentation part of observability,
> it doesn't provide a backend for storing telemetry; you have to forward the 
> data to a backend vendor for storage, alerting, and querying. 
> 
> Prometheus, on the other hand, provides a time-series data store you can use 
> for your metrics, in addition to instrumentation clients. You can view 
> graphs and charts, set up alerts, and query your data via their web user 
> interface. It also encompasses a data format, known as [Prometheus 
> text-based exposition format](https://prometheus.io/docs/instrumenting/exposition_formats/#exposition-formats).